flaeg_test.go

```diff
@@ -2601,3 +2660,41 @@ func TestSplitArgs(t *testing.T) {
 		}
 	}
 }
+
+func TestTypoPrintHelp(t *testing.T) {
+	//init
+	config := &struct {
+		ShortDescription                                                     string `description:"shortDescription"`
+		LoooooooooooooooooooooooooooooooooooooongFieldNameAndLongDescription string `description:"LoooooooooooooooooooooooooooooooooooooongFieldNameAndLongDescription has a very looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong description"`
+		Sh                                                                   string `description:"short"`
+	}{}
+	flagmap := make(map[string]reflect.StructField)
+	err := getTypesRecursive(reflect.ValueOf(config), flagmap, "")
+	if err != nil {
+		t.Fatalf("Error, %s", err.Error())
+	}
+	var stringParser stringValue
+	parsers := map[reflect.Type]Parser{
+		reflect.TypeOf(""): &stringParser,
+	}
+	defaultValmap := map[string]reflect.Value{}
+	getDefaultValue(reflect.ValueOf(config), reflect.ValueOf(config), defaultValmap, "")
+
+	// catch stdout
+	rescueStdout := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+	//test
+	PrintHelp(flagmap, defaultValmap, parsers)
+
+	// read and restore stdout
+	w.Close()
+	out, _ := ioutil.ReadAll(r)
+	os.Stdout = rescueStdout
+	//check
+	const listFlagCheck = `LoooooooooooooooooooooooooooooooooooooongFieldNameAndLongDescription has a very looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong description`
+
+	if strings.Contains(string(out), listFlagCheck) {
```

Comment:
OK, maybe this would have been better to check directly the result of `string(out)` but alright :)